gulp/tasks/default.js

```diff
@@ -1,5 +1,10 @@
-(function() {
-    'use strict';
-    var gulp = require('gulp');
-    gulp.task('default', ['test', 'lint', 'jscs', 'coverage']);
-}());
+'use strict';
+
+var gulp = require('gulp');
+
+gulp.task('default', [
+    'doc-build',
```

Comment:
Are these run in order? If so, does a single task failure stop the whole list of tasks? If so, I think test/lint/jscs should be above doc-build, since it seems pointless to build docs if the test suite is going to fail.